src/components/ClaimMasterPanel.js

```diff
@@ -224,12 +229,12 @@ class ClaimMasterPanel extends FormPanel {
             <Grid item xs={2} className={classes.item}>
               <PublishedComponent
                 pubRef="core.DatePicker"
-                value={edited.dateClaimed}
+                value={edited.dateClaimed ?? new Date()}
```

Comment:
@delcroip @dragos-dobre 
This will set default date to current date, regardless of settings. This was not a part of ticket technically, but it is easier code-wise and I think it makes sense as a initial date for everyone, so I did not changed that. Should this be changed or is it ok right now?

Reply:
Ok for me